profiler/options.go

```diff
@@ -73,8 +74,25 @@ func defaultConfig() *config {
 	for _, t := range defaultProfileTypes {
 		c.addProfileType(t)
 	}
-	// TODO(x): add support for environment variables once we figure out
-	// the naming standards.
+
+	if v := os.Getenv("DD_ENV"); v != "" {
+		WithEnv(v)(&c)
+	}
+	if v := os.Getenv("DD_SERVICE"); v != "" {
```

Comment:
It's unfortunate that people having to set the service name through env. vars will have to use this environment variable, which will result in sort of breaking change in the tracer, given the latest changes. We'll see later on if it turns out to be a bug and might have to add some sort of special hook to work around it.

Reply:
I think it's OK for now. If they set the`DD_SERVICE` env variable, the tracer will behave as it did before if they were using `WithServiceName`.

Reply:
Doesn't `DD_SERVICE` change the service name for some integrations?

Reply:
It *does* do that, but that's a new feature in both the profiler and the tracer. If you're not setting a service, `DD_SERVICE` will change the service name of the tracer and integrations as expected. If you're already setting the service name in the tracer with the deprecated `WithServiceName` though, nothing will change and `DD_SERVICE` will be ignored by the tracer and integrations.

I think the only thing that is *breaking* is if someone wants to set the service in the profiler and *not* the tracer, but the point of unifying the env vars was so that they'd all be used the same way everywhere. Unless I'm missing something else, if it's a breaking change it's breaking by design.